pepper-apis/src/main/java/org/broadinstitute/ddp/export/collectors/PicklistQuestionFormatStrategy.java

```diff
@@ -50,6 +60,28 @@
         props.put("questionType", definition.getQuestionType().name());
         props.put("questionText", HtmlConverter.getPlainText(definition.getPromptTemplate().render("en")));
         props.put("selectMode", definition.getSelectMode());
+        //add PL Groups
+        List<Object> groups = new ArrayList<>();
+        for (PicklistGroupDef group : definition.getGroups()) {
+            Map<String, Object> groupDef = new HashMap<>();
+            String groupStableId = group.getStableId();
+            String groupTxt = HtmlConverter.getPlainText(group.getNameTemplate().render("en"));
+            List<Object> options = new ArrayList<>();
+            for (PicklistOptionDef optionDef : group.getOptions()) {
+                Map<String, String> stableIdTxt = new HashMap<>();
+                stableIdTxt.put("optionStableId", optionDef.getStableId());
+                stableIdTxt.put("optionText", HtmlConverter.getPlainText(optionDef.getOptionLabelTemplate().render("en")));
+                options.add(stableIdTxt);
+            }
+            groupDef.put("groupStableId", groupStableId);
+            groupDef.put("groupText", groupTxt);
+            groupDef.put("options", options);
+            groups.add(groupDef);
+        }
+        if (!groups.isEmpty()) {
```

Comment:
Makes "groups" optional element in elastic json. Will exist only for PL Groups.
wondered if it should be empty list element for non-PL Group questions like groups: []